src/accessors.d

```diff
@@ -22,7 +22,7 @@ struct Write
 
 immutable string GenerateFieldAccessors = `
     mixin GenerateFieldAccessorMethods;
-    mixin(GenerateFieldAccessorMethods);
+    mixin(GenerateFieldAccessorMethods2);
```

Comment:
or the first one to `MixinGenerateFieldAccessorMethods` (or `IncludeGenerateFieldAccessorMethods`), since its only purpose is to insert `static enum GenerateFieldAccessorsMethods` directly into the object.